node/worker/worker.go

```diff
@@ -420,8 +384,8 @@ func New(config *params.ChainConfig, chain *core.BlockChain, engine consensus_en
 		chain:   chain,
 		engine:  engine,
 	}
-	worker.gasFloor = 500000000000000000
-	worker.gasCeil = 1000000000000000000
+	worker.gasFloor = 80000000
```

Comment:
any comments on why change this number?

Reply:
Previous setting doesn't make sense when we first launch mainnet. It was too high. The 80m to 120m limit is 10x that of Ethereum. So we have 10x more capacity than ethereum. And it translate to roughly max 5700 normal txns in a block